### PR TITLE
CTOR-1153 : Refresh cloud-cloudfoundry-api

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-cloudfoundry-api

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 1]
```diff
@@ -1,46 +1,396 @@
...
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Votre collecteur Centreon doit être en mesure d'accéder à votre instance Cloud Foundry (disposer de son url d'hébergement).
+Vous devez disposer d'un compte utilisateur Cloud Foundry ayant les permissions nécessaires pour accéder aux données supervisées via l'API (applications, etc.). 
+Assurez-vous que le serveur Centreon peut communiquer avec l’API Cloud Foundry via le réseau. 
+Autorisez l'accès sortant vers l'URL de l'API (https://api.<domain>) via les ports nécessaires (généralement 443 pour HTTPS).
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:75]:
```suggestion
Autorisez l'accès sortant vers l'URL de l'API (https://api./<domain/>) via les ports nécessaires (généralement 443 pour HTTPS).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 2]
```diff
@@ -1,46 +1,396 @@
...
+title: Cloud Foundry API
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Dépendances du connecteur de supervision
 
-### Centreon Plugin
+Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Cloud Foundry**
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:10]:
```suggestion
Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Cloud Foundry API** 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 3]
```diff
@@ -1,46 +1,396 @@
...
+* [Base Pack](./base-generic.md)
 
-Install this plugin on each needed poller:
+## Contenu du pack
 
-``` shell
-yum install centreon-plugin-Cloud-Cloudfoundry-Api
+### Modèles
+
+Le connecteur de supervision **Cloud Foundry** apporte un modèle d'hôte :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:18]:
```suggestion
Le connecteur de supervision **Cloud Foundry API** apporte un modèle d'hôte :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 4]
```diff
@@ -1,46 +1,396 @@
...
+| instance-state          | N/A   |
+
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Votre collecteur Centreon doit être en mesure d'accéder à votre instance Cloud Foundry (disposer de son url d'hébergement).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:72]:
```suggestion
* Votre collecteur Centreon doit être en mesure d'accéder à votre instance Cloud Foundry (vous devez disposer de son URL d'hébergement).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 5]
```diff
@@ -1,46 +1,396 @@
...
+Vous devez disposer d'un compte utilisateur Cloud Foundry ayant les permissions nécessaires pour accéder aux données supervisées via l'API (applications, etc.).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:73]:
```suggestion
* Vous devez également disposer d'un compte utilisateur Cloud Foundry ayant les permissions nécessaires pour accéder aux données supervisées via l'API (applications, etc.). 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 6]
```diff
@@ -1,46 +1,396 @@
...
+</Tabs>
+
+## Prérequis
+
+Votre collecteur Centreon doit être en mesure d'accéder à votre instance Cloud Foundry (disposer de son url d'hébergement).
+Vous devez disposer d'un compte utilisateur Cloud Foundry ayant les permissions nécessaires pour accéder aux données supervisées via l'API (applications, etc.). 
+Assurez-vous que le serveur Centreon peut communiquer avec l’API Cloud Foundry via le réseau. 
+Autorisez l'accès sortant vers l'URL de l'API (https://api./<domain/>) via les ports nécessaires (généralement 443 pour HTTPS).
+Vérifiez que le certificat SSL de l’API Cloud Foundry est valide et, si nécessaire, ajoutez-le aux certificats approuvés du collecteur Centreon.
+Pour plus d'information, référez-vous à la [documentation officielle](https://docs.cloudfoundry.org/)
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:77]:
```suggestion
* Pour plus d'information, référez-vous à la [documentation officielle Cloud Foundry](https://docs.cloudfoundry.org/).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 7]
```diff
@@ -1,46 +1,396 @@
...
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Votre collecteur Centreon doit être en mesure d'accéder à votre instance Cloud Foundry (disposer de son url d'hébergement).
+Vous devez disposer d'un compte utilisateur Cloud Foundry ayant les permissions nécessaires pour accéder aux données supervisées via l'API (applications, etc.). 
+Assurez-vous que le serveur Centreon peut communiquer avec l’API Cloud Foundry via le réseau.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:74]:
```suggestion
* Assurez-vous que le serveur Centreon peut communiquer avec l’API Cloud Foundry via le réseau. 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 8]
```diff
@@ -1,46 +1,396 @@
...
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Votre collecteur Centreon doit être en mesure d'accéder à votre instance Cloud Foundry (disposer de son url d'hébergement).
+Vous devez disposer d'un compte utilisateur Cloud Foundry ayant les permissions nécessaires pour accéder aux données supervisées via l'API (applications, etc.). 
+Assurez-vous que le serveur Centreon peut communiquer avec l’API Cloud Foundry via le réseau. 
+Autorisez l'accès sortant vers l'URL de l'API (https://api./<domain/>) via les ports nécessaires (généralement 443 pour HTTPS).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:75]:
```suggestion
* Autorisez l'accès sortant vers l'URL de l'API (https://api./<domain/>) via les ports nécessaires (généralement 443 pour HTTPS).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 9]
```diff
@@ -1,46 +1,396 @@
...
+Vérifiez que le certificat SSL de l’API Cloud Foundry est valide et, si nécessaire, ajoutez-le aux certificats approuvés du collecteur Centreon.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:76]:
```suggestion
* Vérifiez que le certificat SSL de l’API Cloud Foundry est valide et, si nécessaire, ajoutez-le aux certificats approuvés du collecteur Centreon.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 10]
```diff
@@ -1,46 +1,396 @@
...
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser une ressource en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib/centreon/plugins/centreon_cloudfoundry_restapi.pl \
+	--plugin=cloud::cloudfoundry::restapi::plugin \
+	--mode=apps-state \
+	--custommode='restapi' \
+	--hostname='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:243]: hostname

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 11]
```diff
@@ -1,46 +1,396 @@
...
+	--api-path='/v2' \
+	--api-username='' \
+	--api-password='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:246]: username and password

File: pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 12]
```diff
@@ -1,46 +1,395 @@
...
 id: cloud-cloudfoundry-api
-title: Cloud Foundry
+title: Cloud Foundry API
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Connector dependencies
+
+The following monitoring connectors will be installed when you install the **Cloud Foundry** connector through the
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:10]:
```suggestion
The following monitoring connectors will be installed when you install the **Cloud Foundry API** connector through the
```

File: pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 13]
```diff
@@ -1,46 +1,395 @@
...
+**Configuration > Monitoring Connector Manager** menu:
+* [Base Pack](./base-generic.md)
+
+## Pack assets
+
+### Templates
+
+The Monitoring Connector **Cloud Foundry** brings a host template:
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:18]:
```suggestion
The Monitoring Connector **Cloud Foundry API** brings a host template:
```

File: pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 14]
```diff
@@ -1,46 +1,395 @@
...
+* **Cloud-Cloudfoundry-Api-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Cloud-Cloudfoundry-Api-custom" label="Cloud-Cloudfoundry-Api-custom">
+
+| Service Alias   | Service Template                              | Service Description      |
+|:----------------|:----------------------------------------------|:-------------------------|
+| Apps-State      | Cloud-Cloudfoundry-Apps-State-Api-custom      | Check applications state |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:29]:
```suggestion
| Apps-State      | Cloud-Cloudfoundry-Apps-State-Api-custom      | Check the state of applications |
```

File: pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 15]
```diff
@@ -1,46 +1,395 @@
...
+| Instances-State | Cloud-Cloudfoundry-Instances-State-Api-custom | Check instances state    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:30]:
```suggestion
| Instances-State | Cloud-Cloudfoundry-Instances-State-Api-custom | Check the state of instances |
```

File: pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 16]
```diff
@@ -1,46 +1,395 @@
...
 
 ## Prerequisites
 
-### Centreon Plugin
+Your Centreon poller must be able to access your Cloud Foundry instance (have its hosting URL).
+You must have a Cloud Foundry user account with the necessary permissions to access the data monitored via the API (applications, etc.).
+Ensure that the Centreon server can communicate with the Cloud Foundry API over the network.
+Allow outbound access to the API URL (https://api./<domain/>) through the required ports (typically 443 for HTTPS).
+Verify that the Cloud Foundry API's SSL certificate is valid and, if necessary, add it to the trusted certificates of the Centreon poller.
+For more informations, refers to the [official documentation](https://docs.cloudfoundry.org/)
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:76]:
```suggestion
* Your Centreon poller must be able to access your Cloud Foundry instance (have its hosting URL ready).
* You must have a Cloud Foundry user account with the necessary permissions to access the data monitored via the API (applications, etc.).
* Ensure that the Centreon server can communicate with the Cloud Foundry API over the network.
* Allow outbound access to the API URL (https://api./<domain/>) through the required ports (typically 443 for HTTPS).
* Check that the Cloud Foundry API's SSL certificate is valid and, if necessary, add it to the trusted certificates of the Centreon poller.
* For more information, refer to the [official Cloud Foundry documentation](https://docs.cloudfoundry.org/)
```

File: pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 17]
```diff
@@ -1,46 +1,395 @@
...
+Once the plugin is installed, log into your Centreon poller's CLI using the
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
+is able to monitor a resource using a command like this one (replace the sample values by yours):
+
+```bash
+/usr/lib/centreon/plugins/centreon_cloudfoundry_restapi.pl \
+	--plugin=cloud::cloudfoundry::restapi::plugin \
+	--mode=apps-state \
+	--custommode='restapi' \
+	--hostname='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:242]: hostname

File: pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md [hunk 18]
```diff
@@ -1,46 +1,395 @@
...
+	--api-path='/v2' \
+	--api-username='' \
+	--api-password='' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:245]: username and password

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md (hunk 11 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:246]:
```suggestion
	--hostname='10.0.0.1' \
	--api-path='/v2' \
	--api-username='XXXX' \
	--api-password='XXXX' \
```

File: pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md (hunk 18 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-cloudfoundry-api.md:245]:
```suggestion
	--hostname='10.0.0.1' \
	--api-path='/v2' \
	--api-username='XXXX' \
	--api-password='XXXX' \
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**